src/main/java/de/tum/in/www1/artemis/repository/ExamUserRepository.java

```diff
@@ -0,0 +1,18 @@
+package de.tum.in.www1.artemis.repository;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import de.tum.in.www1.artemis.domain.exam.ExamUser;
+
+@Repository
+public interface ExamUserRepository extends JpaRepository<ExamUser, Long> {
+
+    @Query("select eu from ExamUser eu where eu.exam.id = :#{#examId} and eu.user.id = :#{#userId}")
+    ExamUser findByExamIdAndUserId(long examId, long userId);
```

Comment:
Please use `@Param` here like in other queries, otherwise the query cannot be executed correctly when testing this locally